ArduPlane/quadplane.cpp

```diff
@@ -1548,6 +1548,7 @@ void QuadPlane::update_transition(void)
     // tailsitters use angle wait, not airspeed wait
     if (is_tailsitter() && transition_state == TRANSITION_AIRSPEED_WAIT) {
         transition_state = TRANSITION_ANGLE_WAIT_FW;
+        transition_initial_pitch= constrain_float(ahrs_view->pitch_sensor,0,-8500);
```

Comment:
You were right to put this where you had it in the first place
(between the else at 1777 and the next if)
but the constraint should be -8500,0 instead of 0,-8500
I hadn't realized it but the state is TRANSITION_ANGLE_WAIT_FW the entire time the vehicle is in FW mode, so this code never gets executed on transition into FW mode.

Reply:
reverted